### PR TITLE
[boschshc] Release v1.1

### PR DESCRIPTION
**Pull request by coeing**
Hi there,

Hope all of you had a good start into the new year :)

We prepared a patch version of our binding, as there were some severe bugs in it which stopped the binding to work after 24 hours and after a restart of the controller.

Plus we use the new version to be up-to-date with the Bosch Rest API (v2.1) and fixed two minor issues (binding title and units used).

Here is the list of changes in this version:
- https://github.com/stefan-kaestle/openhab2-addons/issues/62
- https://github.com/stefan-kaestle/openhab2-addons/issues/72
- https://github.com/stefan-kaestle/openhab2-addons/issues/74
- https://github.com/stefan-kaestle/openhab2-addons/issues/75
- https://github.com/stefan-kaestle/openhab2-addons/issues/77
- https://github.com/stefan-kaestle/openhab2-addons/issues/78

### CODE REVIEW

File: bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschHttpClient.java [hunk 1]
```diff
@@ -194,7 +243,12 @@ public Request createRequest(String url, HttpMethod method) {
      * @return created HTTP request instance
      */
     public Request createRequest(String url, HttpMethod method, @Nullable Object content) {
-        Request request = this.newRequest(url).method(method).header("Content-Type", "application/json");
+        logger.trace("Create request for http client {}", this.toString());
+
+        Request request = this.newRequest(url).method(method).header("Content-Type", "application/json")
+                .header("api-version", "2.1") // see https://github.com/BoschSmartHome/bosch-shc-api-docs/issues/46
```
Comment (cpmeister) [inline bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschHttpClient.java:249]: Would this cause any problems with devices using older apis?

File: bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/i18n/boschshc.properties [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+offline.conf-error-empty-ip = No network address set.
+offline.conf-error-empty-password = No system password set.
+offline.conf-error-offline = The Bosch Smart Home Controller is offline or network address is wrong.
 offline.conf-error-pairing = Press pairing button on the Bosch Smart Home Controller.
-offline.not-reachable = Smart Home Controller is not reachable.
+offline.not-reachable = The Bosch Smart Home Controller is not reachable.
 offline.conf-error-ssl = The SSL connection to the Bosch Smart Home Controller is not possible.
+offline.long-polling-failed.http-client-null = Long polling failed and could not be restarted because http client is null.
+offline.long-polling-failed.trying-to-reconnect = Long polling failed, will try to reconnect.
+offline.interrupted = Conneting to Bosch Smart Home Controller was interrupted.
```
Comment (cpmeister) [inline bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/i18n/boschshc.properties:12]: Please add a newline.

**Review by cpmeister — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschHttpClient.java (hunk 1 above)
Comment (coeing) [reply bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschHttpClient.java:249]: Not as far as I know @GerdZanker Did you read any information about this?

File: bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/i18n/boschshc.properties (hunk 2 above)
Comment (GerdZanker) [reply bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/i18n/boschshc.properties]: Done

File: bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschHttpClient.java (hunk 1 above)
Comment (GerdZanker) [reply bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschHttpClient.java:249]: Thank you @cpmeister for sharing your experience regarding API compatibility.

The update to 2.1 was rolled out in Dezember 2020 and is installed automatically within weeks. I assume that we not run into issues after more than two months. 

Statement from the BoschSmartHome maintainers is:

>Only very, very few might not have API 2.1 available by now

The actual supported api version is also logged during binding initialization and therefore we can detect old Smart Home Controllers and suggest to do an update.

**Review by cpmeister — APPROVED** [review]

**Pull request merged**